criterion-measurement/src/Criterion/Measurement/Types.hs

```diff
@@ -125,6 +125,8 @@ data Measured = Measured {
 
     , measAllocated          :: !Int64
       -- ^ __(GC)__ Number of bytes allocated.  Access using 'fromInt'.
+    , measPeakMbAllocated    :: !Int64
```

Comment:
It would definitely be worth noting the addition of this new field to the `changelog`. Moreover, it is also worth noting that the behavior of `Measured`'s `Binary` and `ToJSON` instances have changed, since it's possible that there are user programs that deserialize `Measured` values directly from binary or JSON. (I don't think that rises to the level of needing a major version bump or anything, but it is worth at least a mention.)

Reply:
Good point. Actually, I think this probably _does_ require a major version change for `criterion-measurement` and `criterion` because both export `Measured`, which has the new `measPeakMbAllocated` field.

Reply:
Ah, I had forgotten that `Measured` is exported. In that case, I agree that major version bumps are in order.